### PR TITLE
Refine/add `scissors`[`-square`] (snippet) icon

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #1068.

`scissors-line-dashed` alts:

preview

### CODE REVIEW

**Comment by jguddas** [comment]
You might not be able to cut with them, but they do look cute and that's all that counts.

Don't know if the square version works with 3px stroke width 🤔

**Comment by jguddas** [comment]
gap could be smaller image

**Comment by danielbayley** [comment]
> gap could be smaller

I’m pretty sure the gap was a violation before… I opted to prioritise snapping to the grid, should I not?

**Comment by jguddas** [comment]
> I opted to prioritise snapping to the grid, should I not?

@karsa-mistmere what do you think?

My feeling tends more towards snapping.

**Review by ericfennis — APPROVED** [review]
@danielbayley Love these ones!
Nice work

**Comment by danielbayley** [comment]
See also latest `push` https://github.com/lucide-icons/lucide/pull/1266#issue-1706498056

**Comment by karsa-mistmere** [comment]
> > I opted to prioritise snapping to the grid, should I not?
> 
> @karsa-mistmere what do you think?
> 
> My feeling tends more towards snapping.

Ah yes, the ever so present negative space issue we keep running into. 😆 

image 
image

Column no.1: snapping to grid
Column no.2: exactly 2px gap
Column no.3: exactly 1px gap
Column no.4: current version

Rows are at 1, 2 and 3px stroke width respectively.

I think that:
- at 1px stroke width col.3 looks best.
- at 2px stroke width col.4 does.
- at 3px stroke width col.1 does

Therefore: I think we should stick with col.2 where all three look equally mediocre. :D

**Comment by danielbayley** [comment]
> a gap of exactly 2px as opposed to snapping to the grid

@karsa-mistmere Done.

File: icons/scissors.svg [hunk 1]
```diff
@@ -10,8 +10,8 @@
   stroke-linejoin="round"
 >
   <circle cx="6" cy="6" r="3" />
+  <path d="M8.2 8.2 12 12" />
+  <path d="M14.8 14.8 20 20" />
   <circle cx="6" cy="18" r="3" />
-  <line x1="20" x2="8.12" y1="4" y2="15.88" />
-  <line x1="14.47" x2="20" y1="14.48" y2="20" />
-  <line x1="8.12" x2="12" y1="8.12" y2="12" />
+  <path d="M20 4 8.2 15.8" />
```
Comment (karsa-mistmere) [inline icons/scissors.svg:16]: There seems to be an issue with optimisation:

Original outline:
image

New outline:
image
(note gaps)

File: icons/scissors-line-dashed.svg [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  stroke-linejoin="round"
+>
+  <path d="M5.4 9.4 8 12" />
+  <circle cx="4" cy="8" r="2" />
+  <path d="m14 6-8.6 8.6" />
+  <circle cx="4" cy="16" r="2" />
+  <path d="M10.8 14.8 14 18" />
+  <path d="M16 12h-2" />
+  <path d="M22 12h-2" />
+</svg>
```
Comment (karsa-mistmere) [inline icons/scissors-line-dashed.svg:19]: All of these could also do with an extra decimal of precision:
image

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

File: icons/scissors-square-dashed-bottom.svg
Comment (karsa-mistmere) [inline icons/scissors-square-dashed-bottom.svg:1]: This one still has some misaligned paths.

File: icons/scissors-square.svg
Comment (karsa-mistmere) [inline icons/scissors-square.svg:1]: Also has some misaligned paths.

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**